components/checkbox.module.css

```diff
@@ -0,0 +1,22 @@
+.checkbox {
+  position: relative;
+  height: 24px;
+  width: 24px;
+  border: 1px solid #d1d6dc;
+  border-radius: 4px;
+
+  &:after {
```

Comment:
windicss has an `:after` variant for targeting this https://windicss.org/utilities/variants.html#pseudo-elements